pkg/ovsdb/transact.go

```diff
@@ -980,7 +983,7 @@ func doInsert(txn *Transaction, ovsOp *libovsdb.Operation, ovsResult *libovsdb.O
 		for _, row := range txn.cache.Table(txn.request.DBName, ovsOp.Table) {
 			if (*row)[COL_UUIDNAME] == ovsOp.UUIDName {
```

Comment:
UUIDName can't be part of the cache, and the key, it is not UUID, and is provided only to  connect to different entries in the request .

Your logic here is wrong, you first, bring entire table, but when did not check the new UUID with the  cache